dev-infra/commit-message/parse.ts

```diff
@@ -6,80 +6,51 @@
  * found in the LICENSE file at https://angular.io/license
  */
 
-import {exec} from '../utils/shelljs';
+import {Commit, Options, sync as parse} from 'conventional-commits-parser';
 
-/** A parsed commit message. */
-export interface ParsedCommitMessage {
-  header: string;
-  body: string;
-  bodyWithoutLinking: string;
-  type: string;
-  scope: string;
-  subject: string;
-  isFixup: boolean;
-  isSquash: boolean;
-  isRevert: boolean;
-}
+import {exec} from '../utils/shelljs';
 
 /** Regex determining if a commit is a fixup. */
 const FIXUP_PREFIX_RE = /^fixup! /i;
-/** Regex finding all github keyword links. */
-const GITHUB_LINKING_RE = /((closed?s?)|(fix(es)?(ed)?)|(resolved?s?))\s\#(\d+)/ig;
 /** Regex determining if a commit is a squash. */
 const SQUASH_PREFIX_RE = /^squash! /i;
 /** Regex determining if a commit is a revert. */
 const REVERT_PREFIX_RE = /^revert:? /i;
-/** Regex determining the scope of a commit if provided. */
-const TYPE_SCOPE_RE = /^(\w+)(?:\(([^)]+)\))?\:\s(.+)$/;
-/** Regex determining the entire header line of the commit. */
-const COMMIT_HEADER_RE = /^(.*)/i;
-/** Regex determining the body of the commit. */
-const COMMIT_BODY_RE = /^.*\n\n([\s\S]*)$/;
+
+enum NoteSections {
+  BREAKING_CHANGE = 'BREAKING_CHANGE',
```

Comment:
Shouldn't this be:

```suggestion
  BREAKING_CHANGE = 'BREAKING CHANGE',
```

(Maybe we should have some tests for these note sections :wink:)

Reply:
Whoops